en/components/date_range_picker.md

```diff
@@ -0,0 +1,428 @@
+
+Start and end date are separated by `-` when shown in the component's read-only input. When defining different inputs for start and end, they are separated by the `to` preposition. The latter is localizable and can be overwritten with a template. Follow the [Templating](#templating) example below, for more details.
+
+#### Projection of child components
+
+The Date Range Picker component allows projecting the following child components: [`igxLabel`](label_input.md),[`IgxHint`]({environment:angularApiUrl}/classes/igxhintdirective.html), `igxPrefix`, `igxSuffix` and `igxPickerToggle`.
+
+```html
+<igx-date-range-picker #dateRangePicker="ngModel" [(ngModel)]="range">
+    <label igxLabel>Flight dates</label>
+    <igx-hint *ngIf="dateRangePicker.invalid">
+        Please choose start and end date!
+    </igx-hint>
+</igx-date-range-picker>
+```
+
+When two seperate inputs are defined this is done using the `IgxDateRangeStartComponent` and `IgxDateRangeEndComponent`, which are extending the existing `IgxInputGroupComponent`. This means they also can contain and can project `IgxInput`, [`igxLabel`](label_input.md),[`IgxHint`]({environment:angularApiUrl}/classes/igxhintdirective.html), `igxPrefix` and `igxSuffix`. In order to have date functionalities, the projected `IgxInput` should be decorated with [IgxDateTimeEditorDirective]({environment:angularApiUrl}/classes/igxdatetimeeditordirective.html) directive, as shown in the code snippet below:
+
+```html
+<igx-date-range-picker [(ngModel)]="range">
+    <igx-date-range-start>
+        <input igxInput igxDateTimeEditor type="text">
+        <igx-picker-toggle igxPrefix>
+            <igx-icon>calendar_today</igx-icon>
+        </igx-picker-toggle>
+    </igx-date-range-start>
+    <igx-date-range-end>
+        <input igxInput igxDateTimeEditor type="text">
+    </igx-date-range-end>
+</igx-date-range-picker>
+```
+
+### Examples
+
+#### Label
+To define a label for the range date picker the [`igxLabel`](label_input.md) directive should be used, like shown in the snippet below:
+
+```html
+<igx-date-range-picker [(ngModel)]="range">
+    <label igxLabel>Flight dates</label>
+</igx-date-range-picker>
+```
+
+For more usage of projected components check the [Application Demo](#application-demo) at the end, where [`IgxHint`]({environment:angularApiUrl}/classes/igxhintdirective.html) is used.
+
+#### Calendar icon
+By default, in a single read-only mode, the default calendar icon is provided as an prefix. For developer ease, the calendar icon itself can be changed or newly defined using the `igxPickerToggle` component. It can be decorated with either `igxPrefix` or `igxSuffix`, which will define its position - either at the start of the input or at the end. For changing the default icon position and putting the icon as a suffix the following code can be used:
```

Comment:
```suggestion
By default, in a single read-only mode, the default calendar icon is provided as a prefix. For development ease, the calendar icon itself can be changed or newly defined using the `igxPickerToggle` component. It can be decorated with either `igxPrefix` or `igxSuffix`, which will define its position - either at the start of the input or at the end. For changing the default icon position and putting the icon as a suffix the following code can be used:
```